cpp/src/Slice/Grammar.y

```diff
@@ -1453,7 +1450,7 @@ interface_list
 | ICE_VALUE
 {
     currentUnit->error("illegal inheritance from type Value");
-    $$ = make_shared<ClassListTok>(); // Dummy
```

Comment:
This is the only place where we use `ClassListTok` and it makes no sense.
The name of this grammar rule is `interface_list`, and the caller immediately casts this to an `InterfaceListTok` anyways.